As far as I understand, this was fixed in the latest version of SAP, the current locked version fails to build on Windows with these kind of errors:

```
swift-argument-parser\Sources\ArgumentParser\Usage\HelpGenerator.swift:299:13: error: cannot find type 'CONSOLE_SCREEN_BUFFER_INFO' in scope
```

Would you like me to bring back the SAP update to this PR to fix that?

--- a/Package.resolved
+++ b/Package.resolved
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/apple/swift-argument-parser",
         "state": {
           "branch": null,
-          "revision": "eb51f949cdd0c9d88abba9ce79d37eb7ea1231d0",
-          "version": "0.2.0"
+          "revision": "6b2aa2748a7881eebb9f84fb10c01293e15b52ca",
+          "version": "0.5.0"
         }
       }
     ]

--- a/Package.swift
+++ b/Package.swift
@@ -24,7 +24,7 @@ let package = Package(
             targets: ["Benchmark"])
     ],
     dependencies: [
-        .package(url: "https://github.com/apple/swift-argument-parser", from: "0.0.1"),
+        .package(url: "https://github.com/apple/swift-argument-parser", from: "0.5.0"),
     ],
     targets: [
         .target(

